cli/compose/schema/bindata.go

```diff
@@ -155,8 +155,8 @@ func dataConfig_schema_v33Json() (*asset, error) {
 // It returns an error if the asset could not be found or
 // could not be loaded.
 func Asset(name string) ([]byte, error) {
-	cannonicalName := strings.Replace(name, "\\", "/", -1)
-	if f, ok := _bindata[cannonicalName]; ok {
+	canonicalName := strings.Replace(name, "\\", "/", -1)
+	if f, ok := _bindata[canonicalName]; ok {
```

Comment:
These changes are in a file that is generated. It should have caused CI to fail. I'm not sure why it passed, I'll have to look into it.

The changes in this file should be reverted.